content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -86,7 +94,7 @@ The :guilabel:`Price Rules` tab functionality on a pricelist form varies dependi
 However, the :guilabel:`Time-based rules` tab and :guilabel:`Configuration` tab are always the same,
 regardless of the chosen :guilabel:`Pricelists` setting.
 
-Price rules tab (multiple prices per product)
+Price Rules tab (multiple prices per product)
```

Comment:
Don't need the parent heading label here. 
Since it's an h4 it's implied that we're still under the Price Rules tab h3

```suggestion
Multiple prices per product
```